proto/process/agent.proto

```diff
@@ -465,3 +472,35 @@ message Host {
 message DNSEntry {
   repeated string names = 1;
 }
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto
+message Pod {
+	string name = 1;
+	string namespace = 2;
+	int64 startTime = 3;
+	string IP = 4;
+	string nominatedNodeName = 5;
+	string nodeName = 6;
+	int32 restartCount = 7;
+	repeated ContainerStatus containerStatuses = 8;
+	repeated string labels = 9;
```

Comment:
Are `labels` and `tags` the same thing or different? I currently see both in here.

Reply:
yes `labels` are the pod labels and `tags` are our internal tags stored in the tagger for this particular pod object some info will be duplicated with what we collect from pod already but some will not (ie `kube_service`)

Reply:
I see. Maybe put a comment on tag to mark it for internal use so people don't ask this question again.